### ISSUE TITLE
Support okhttp client

### ISSUE THREAD

**Issue by vprithvi** [uber/jaeger-client-java#85]
Use [okhttp](http://square.github.io/okhttp/) [interceptors](https://github.com/square/okhttp/wiki/Interceptors) to add tracing support

**Comment by pavolloffay** [comment]
see https://github.com/opentracing-contrib/java-okhttp. It does not support jaeger's in-process propagation, but that will be handled by OT.

**Comment by yurishkuro** [comment]
@pavolloffay +1, once active span manager lands in OT, we should revisit all instrumentations in this repo and possibly decommission some of them in favor of ot-contrib.

**Comment by yurishkuro** [comment]
Closing in favor of #182
